satpy/etc/enhancements/generic.yaml

```diff
@@ -1289,3 +1289,18 @@ enhancements:
   image_ready:
     standard_name: image_ready
     operations: []
+
+  mw183_humidity:
+    # matches AWS
+    standard_name: mw183_humidity
+    operations:
+    - name: stretch
+      method: !!python/name:satpy.enhancements.stretch
+      kwargs:
+        stretch: crude
+        min_stretch: [290, 290, 290]
+        max_stretch: [190, 190, 190]
+    - name: gamma
+      method: !!python/name:satpy.enhancements.gamma
+      kwargs:
+        gamma: [1.5, 1.2, 1.2]
```

Comment:
No need for this, AWS/MWS enhancement is already defined. Oh, but why this definition is different to that one?

Reply:
Yes, this is a new, and I think, better recipe for the MW humidity RGBs.

Reply:
If this one is better, the one specified specifically for MWR should be deleted. It will be used instead of this one because it is defined in the `<instrument>.yaml` file.

Reply:
The reason most likely is that the enhancement file was still named as `aws.yaml` and not `mwr.yaml` so the name of the file and the sensor didn't match.